torch/csrc/jit/ir.cpp

```diff
@@ -520,6 +520,10 @@ std::shared_ptr<Graph> Graph::copy() {
   return new_g;
 }
 
+bool Value::isNone() const {
```

Comment:
This is dangerously named. `if(!v->isNone())` seems to say that v cannot be none. 
Maybe rename: `mustBeNone()`